setup.py

```diff
@@ -16,6 +16,7 @@
     long_description_content_type="text/markdown",
     url="https://github.com/dainstudios/untidy",
     install_requires=["pandas"],
```

Comment:
how about numpy and seaborn (for the demo) ?

Reply:
divided requirements for that